framy_generator/lib/generator/story_board_generator.dart

```diff
@@ -0,0 +1,80 @@
+import 'package:framy_generator/framy_object.dart';
+import 'package:framy_generator/generator/utils.dart';
+import 'package:framy_generator/generator/widget_dependency_utils.dart';
+
+String generateStoryBoardPage(List<FramyObject> widgetFramyObjects, List<FramyObject> modelFramyObjects) {
+  String gridViewChildren = widgetFramyObjects.fold(
+    '',
+        (previousValue, element) => previousValue + _generateStoryboardWidgetPage(element),
+  );
+  return '''
+class FramyStoryBoardPage extends StatelessWidget {
+  const FramyStoryBoardPage() : super(key: const Key('FramyStoryBoardPage'));
+
+  @override
+  Widget build(BuildContext context) {
+    return GridView.extent(
+      padding: const EdgeInsets.all(16),
+      maxCrossAxisExtent: 300,
+      crossAxisSpacing: 16,
+      mainAxisSpacing: 16,
+      childAspectRatio: 1 / 2,
+      children: [
+      $gridViewChildren
+      ],
+    );
+  }
+}
+
+class FramyStoryboardCustomPageWithDependencies extends StatelessWidget {
+  final List<FramyDependencyModel> dependencies;
+  final Widget Function(DependencyValueGetter dependencyValueGetter) builder;
+
+  const FramyStoryboardCustomPageWithDependencies(
+      {Key key, this.dependencies, this.builder})
+      : super(key: key);
+
+  FramyDependencyModel dependency(String name) =>
+      dependencies.singleWhere((d) => d.name == name);
+
+  dynamic dependencyValue(String name) => dependency(name).value;
+
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      decoration: BoxDecoration(
+        border: Border.all(color: Colors.black54),
+        borderRadius: BorderRadius.circular(16),
+      ),
+      child: builder(dependencyValue),
+    );
+  }
+}
+''';
+}
+
+String _generateStoryboardWidgetPage(FramyObject framyObject) {
```

Comment:
It must be the same piece of code as in custom widget page generator (no duplicate)